sympy/physics/mechanics/kane.py

```diff
@@ -784,9 +910,26 @@ def auxiliary_eqs(self):
 
     @property
     def mass_matrix(self):
-        """The mass matrix of the system."""
+        """The mass matrix of the following dynamic differential equations::
+
+            M * u' = f
+
+        where u contains the independent and dependent generalized speeds::
+
+            u = [u_ind]
+                [u_dep]
+
+        The equations can be formed with these class attributes::
+
+            >>> lhs = mass_matrix * generalized_accelerations
+            >>> rhs = forcing
+            >>> Eq(lhs, rhs)
+
+        """
         if not self._fr or not self._frstar:
-            raise ValueError('Need to compute Fr, Fr* first.')
+            msg = ('kanes_equations() must be run before the mass matrix is '
+                   'available.')
+            raise ValueError(msg)
```

Comment:
```
if not self._fr or not self._frstar:
             raise ValueError('Need to compute Fr, Fr* first.')
```
This line also appears in mass_matrix_full, forcing and forcing full. I think it should receive the same treatment you gave `none_handler` where it is factored into a function call to reduce code redundancy. Would such a function be better as a method of `KanesMethod` or at the top of the module like `_none_handler`?